src/veil/zerocoin/ztracker.h

```diff
@@ -55,13 +53,6 @@ class CzTracker
     bool UpdateZerocoinMint(const CZerocoinMint& mint);
     bool UpdateState(const CMintMeta& meta);
     void Clear();
-    mutable CCriticalSection cs_modify_lock;
```

Comment:
when resolving the conflict with ztracker.h; note that the conflict is due to https://github.com/Veil-Project/veil/commit/0a0a1159dd596e2337c62f2c74194bd59ac7d527 adding the `cs_remove_pending` critical section.  so that needs to be preserved here.

Reply:
I'd rather we move cs_remove_pending to just above UpdateStatusInternal  since it is used internally there.
Arguments are: it is polluting the class public space and this is meant for local usage.
Can I?

Reply:
on second thought let's keep the remove lock in the declaration but move it to the private area.
Good?

Reply:
Thank you for asking.  Excellent question.  And if this was in a code review for #657 I would welcome the idea.

However this is just making sure the conflict is resolved.   As such, we really need to keep this PR focused to what this PR is for, and we can follow up with a refactoring PR that addresses the pollution of the class public space (as I'm sure there are other places were improvements under that title can be made).

So for now, merge where #657 put it; and if you want to throw in an issue for refactoring public/private space and reference it.. by all means do so.  Then we can look at that then.